src/code.py

```diff
@@ -0,0 +1,56 @@
+import time
+import board
+import neopixel
+
+import json
+
+# Orange is modified here to make it more distinct from yellow
+corners = [0x00ff00, 0xff3300, 0xff00ff, 0xffff00]
+
+PIXELS = neopixel.NeoPixel(board.NEOPIXEL, 4)
+PIXELS.brightness = 0.02  # The LEDs are very bright
+
+
+def set_corner(corner, arena=None):
+    "Set LEDs 1/2 to the corner's colour, the LED set depends on the arena"
```

Comment:
Question: How are we intending to tell which device is which without plugging them into power? Looking at the photos, it doesn't look like there's much of a keyring attachment.

Reply:
Heat shrink with a piece of card on the bottom side would work nicely and help protect the device